tensorboard/compat/proto/summary.proto

```diff
@@ -53,8 +53,32 @@ message SummaryMetadata {
 
   // Longform readable description of the summary sequence. Markdown supported.
   string summary_description = 3;
+
+  // Class of data stored in this time series. Required for compatibility with
```

Comment:
Note: This file is meant to be kept in sync with TensorFlow’s copy, so
it’s not great to be introducing manual changes here. Once this file is
reviewed and approved (but not necessarily merged), I’ll make a
corresponding change to TensorFlow and get that submitted. My
understanding is that we can be fairly comfortable making this change
given that TensorFlow no longer exposes the protos from its Python API
and to avoid guaranteeing any API compatibility. Please correct me if
anything here sounds off.

Reply:
Changes merged upstream into TensorFlow, so this is now ready:
<https://github.com/tensorflow/tensorflow/commit/ac6248d02be521dddddc1be1e2cd70e75d7e0c4b>